neuralpp/test/quick_tests/symoblic/simple_test.py

```diff
@@ -73,15 +76,15 @@ def test_gotchas_xor_and_divide() -> None:
     assert Rational(1, 3) == Integer(1)/Integer(3)
 
 
-def test_substitution():
+def test_substitution() -> None:
```

Comment:
Why did you add "-> None"? Was that required?

Reply:
No. Just want to write it in a more typed way..

Reply:
Yes, I think it may be distracting since the rest of the code is not typed.

Reply:
Got rid of `-> None` in test files.